payment/transport.go

```diff
@@ -0,0 +1,62 @@
+package payment
+
+import (
+	"encoding/json"
+	"net/http"
+	"github.com/go-kit/kit/log"
+	httptransport "github.com/go-kit/kit/transport/http"
+	"github.com/gorilla/mux"
+	"golang.org/x/net/context"
+)
+
+// MakeHTTPHandler mounts the endpoints into a REST-y HTTP handler.
+func MakeHTTPHandler(ctx context.Context, e Endpoints, logger log.Logger) http.Handler {
+	r := mux.NewRouter().StrictSlash(false)
+	options := []httptransport.ServerOption{
+		httptransport.ServerErrorLogger(logger),
+		httptransport.ServerErrorEncoder(encodeError),
+	}
+
+	r.Methods("GET").Path("/paymentAuth").Handler(httptransport.NewServer(
+		ctx,
+		e.AuthoriseEndpoint,
+		decodeAuthoriseGetRequest,
+		encodeAuthoriseGetResponse,
```

Comment:
Why do you call it xxcodeAuthoriseGetRexxx, instead of just xxcodeAuthoriseRexxx?